### PR TITLE
Start cloud8 development

### PR DESCRIPTION
**Pull request by dirkmueller**
Add cloudsource=susecloud8/develcloud8 support, remove susecloud7
leftovers.

### CODE REVIEW

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -1242,11 +1250,18 @@ function onadmin_set_source_variables
...
         ;;
-        susecloud7)
-            CLOUDSLE12DISTPATH=/ibs/SUSE:/SLE-12-SP2:/Update:/Products:/Cloud7/images/iso/
-            CLOUDSLE12DISTISO="SUSE-OPENSTACK-CLOUD-7-$arch*Media1.iso"
-            CLOUDSLE12TESTISO="CLOUD-7-TESTING-$arch*Media1.iso"
-            CLOUDLOCALREPOS="SUSE-OpenStack-Cloud-7-official"
+        develcloud8)
+            CLOUDSLE12DISTPATH=/ibs/Devel:/Cloud:/8/images/iso
+            [ -n "$TESTHEAD" ] && CLOUDSLE12DISTPATH=/ibs/Devel:/Cloud:/8:/Staging/images/iso
+            CLOUDSLE12DISTISO=${want_cloud7_iso:="SUSE-OPENSTACK-CLOUD-8-$arch*Media1.iso"}
```
Comment (jsuchome) [inline scripts/qa_crowbarsetup.sh]: `want_cloud7_iso` ?

File: scripts/qa_crowbarsetup.sh [hunk 2]
```diff
@@ -403,6 +406,14 @@ function addcloud7pool
         "cloudpool"
 }
 
+function addcloud8pool
+{
+    add_mount "SUSE-OpenStack-Cloud-8-Pool" \
+        "$nfsserver_ip:$nfsserver_base_path/repos/$arch/SUSE-OpenStack-Cloud-8-Pool/" \
+        "$tftpboot_repos12sp2_dir/SUSE-OpenStack-Cloud-8-Pool/" \
```
Comment (rsalevsky) [inline scripts/qa_crowbarsetup.sh]: 12sp2?

File: scripts/qa_crowbarsetup.sh [hunk 3]
```diff
@@ -816,7 +834,13 @@ function onadmin_prepare_sles12plus_cloud_repos
 
     local sles12optionalrepolist
     local targetdir
-    if iscloudver 7plus; then
+    if iscloudver 8plus; then
+        sles12optionalrepolist=(
+            SUSE-OpenStack-Cloud-8-Pool
+            SUSE-OpenStack-Cloud-8-Updates
+        )
+        targetdir="$tftpboot_repos12sp2_dir"
```
Comment (jsuchome) [inline scripts/qa_crowbarsetup.sh]: Also here

File: scripts/qa_crowbarsetup.sh [hunk 4]
```diff
@@ -886,6 +910,22 @@ function onadmin_prepare_sles12sp2_installmedia
...
+function onadmin_prepare_sles12sp3_installmedia
+{
+    local a
+    for a in $architectures; do
+        local sles12sp3_mount="$tftpboot_suse12sp3_dir/$a/install"
+        add_mount "SLE-12-SP3-Server-TEST/sle-12-$a" \
+            "$nfsserver_ip:$nfsserver_base_path/suse-12.3/$a/install" \
+            "$sles12sp2_mount"
+
+        if [ ! -d "$sles12sp2_mount/media.1" ] ; then
```
Comment (jsuchome) [inline scripts/qa_crowbarsetup.sh]: sles12sp2_mount

**Comment by dirkmueller** [comment]
(still ironing out last kinks)

**Comment by dirkmueller** [comment]
ok, tested successfully

**Comment by jsuchome** [comment]
Looks good - but for full successful installation we need modifications in `install-chef-suse.sh` - do we want to wait for modifications there? ... Hm, probably not.

**Pull request merged**